docs/basics/integrations/slack.md

```diff
@@ -2,28 +2,92 @@
 id: slack
 title: Slack Integration
 sidebar_label: Slack
-description: Link your Sauce Labs account with your Slack workspace to set up auto-notification of your test results.
+description: Link your Sauce Labs account with your Slack workspace set up auto-notification of your test results.
 keywords:
 - share-test-results
 - slack
 - how-to
 ---
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <p><span className="sauceDBlue">COMING SOON</span></p>
 
 The Sauce Labs Slack app allows you to send test result notifications to selected workspace channels.
 
-<!-- ## What You'll Need
+## What You'll Need
 
 * A Sauce Labs account (if you don't have one, start a [free trial](https://saucelabs.com/sign-up))
 * Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
-* A [Slack](https://slack.com/) workspace with Admin rights
+* A [Slack](https://slack.com/) workspace
 
-## Install the App
+## Install the Slack Plugin
```

Comment:
I guess it's a slack app, not a plugin. What do you think?

Reply:
Yeah. It's weird cause there are 2 parts to it, right? I have been thinking about it as:
- The "App" is the part that needs to be added to the workspace. This brings up a good point because I have actually been curious about how that happens. As a Sauce Labs user, I don't have to do anything to 1) see the notifications in my workspace; or 2) add the App to a channel in my workspace. 
- The "Plugin" is the part that an individual user installs to be able to be recognized as the "owner" if I run a test that has slack notifications configured. 
Do I understand this correctly? I would actually like to be clearer on this...maybe we can have a quick conversation in slack (ha!)